pysteps/io/importers.py

```diff
@@ -400,14 +401,14 @@ def import_mrms_grib(filename, extent=None, window_size=4, **kwargs):
         unit="mm/h",
         transform=None,
         zerovalue=0,
-        projection=proj_params,
+        projection=proj_def.strip(),
         yorigin="upper",
         threshold=_get_threshold_value(precip),
         x1=x1,
         x2=x2,
         y1=y1,
         y2=y2,
-        cartesian_unit="m"
+        cartesian_unit="m",
```

Comment:
Should this be 'degrees' by the way?

Reply:
Yes,  since 
```
x1, y1 = pr(ul_lon, lr_lat)
x2, y2 = pr(lr_lon, ul_lat)
```
returns x and y in degrees (no projection).